pkg/parser/uri_test.go

```diff
@@ -0,0 +1,68 @@
+package parser
+
+import (
+	"testing"
+)
+
+type tests struct {
+	dsn    string
+	scheme string
+	path   string
+	query  string
+}
+
+var fTests = []tests{
+	{"", "", "", ""},
+	{"file:///some/path/to/file.csv?format=csv", "file", "/some/path/to/file.csv", "format=csv"},
+	{"file:///some/path/to/file.csv", "file", "/some/path/to/file.csv", ""},
+	{"os:stdout?format=csv", "os", "", "format=csv"},
+}
```

Comment:
Could you add a test for the following URI's
- `file://test.csv`
- `file:///root/test.csv`

Reply:
What will be the expected outputs for `file://test.csv`?
We already have a [test case](https://github.com/hellofresh/klepto/blob/cf881804f48ce642889f8a8f21018d9a73b33f84/pkg/parser/uri_test.go#L17) for `file:///root/test.csv`